tests/pipeline/test_engine.py

```diff
@@ -146,10 +149,23 @@ def compute(self, today, assets, out, *inputs):
         out[:] = sum(inputs).sum(axis=0)
 
 
-class ConstantInputTestCase(TestCase):
+class ConstantInputTestCase(WithTradingEnvironment, ZiplineTestCase):
+    asset_ids = 1, 2, 3
```

Comment:
These feel like they want to be an argument (i.e. an overridable class-level attribute) to `WithAssetFinder`.  Right now, this test is relying on the fact that these are the default sids for `tmp_asset_finder`, which is pretty far away from here.